sdk/core/Azure.Core.TestFramework/README.md

```diff
@@ -251,6 +251,8 @@ test-proxy push -a <path-to-assets.json>
 
 On completion of the push, a newly created tag will be stamped into the `assets.json` file. This new tag must be committed and pushed to your package directory along with any other changes.
 
+☑️ Permission is required for updating the assets repository. If you failed when executing `test-proxy push` command, please [join a partner write team](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/785/Externalizing-Recordings-(Asset-Sync)?anchor=permissions-to-%60azure/azure-sdk-assets%60).
```

Comment:
What's the reason for using this icon?  I don't see this convention used elsewhere in the doc, and this doesn't really highlight and call attention to this, in my opinion.  It seems more appropriate for defining a checklist.

I'd rather see us annotate with **NOTE:** and/or some kind of bit exclamation point.

Reply:
Thanks @jsquire I updated to **NOTE**.